src/content/whats-new/2024/06/whats-new-06-11-click-to-parse.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Streamline log parsing'
+summary: 'Use our new options to generate ingest parsing or query time parsing rules'
+releaseDate: '2024-06-11'
+learnMoreLink: 'https://newrelic.com/blog/how-to-relic/click-to-parse'
+getStartedLink: 'https://youtu.be/tvK6MlkvD6Y?si=5klygPBzQyY5qsI8'
+---
+
+While the majority of valuable parsing information is automatically gathered during the ingestion of logs into the New Relic platform, building additional queries can be labor-intensive. Logs parsing just became easier for both types of parsing at New Relic: 
+
+* Ingest parsing: These are rules that create permanent changes in your logs and are based on Grok rules. In the past, you had to create your own Grok rules, but now we can create these complex rules for you with our NRAI assistant.  
+* Query time parsing: We generate temporary parsing rules written in NRQL as you make log selections in our UI.
+
+## Ingest parsing with AI assistance
+
+We are expanding our public preview for our AI-powered ingest to allow you to parse your logs using automatically generated Grok rules. New Relic AI can help you create and troubleshoot parsing rules by:
+
+* **Suggesting parsing rules**: When you create a parsing rule in the UI, ask NRAI to create the parsing rule for you. 
+* **Explaining parsing rules**: If you don't understand the parsing rule our AI suggests, you can ask AI to explain the rule and suggest an even more efficient one if available. 
+
+![A screenshot showing NRAI enabled.](./images/pic1.png "A screenshot showing NRAI enabled.")
+
+<figcaption>Ask AI to help you generate Grok patterns for ingest parsing.</figcaption>
```

Comment:
```suggestion
<figcaption>Ask New Relic AI to help you generate New Relic AI patterns for ingest parsing.</figcaption>
```

Reply:
a little repetitive/redundant but def want to remove mentions of grok

Reply:
This is actually Grok! It tripped me up too. It's a logs thing.